### PR TITLE
python@3.11.1: Fix hash

### LINKED ISSUE #86: python@3.11.1: hash check failed
Checking hash of install-pep-514.reg ... ERROR Hash check failed!
App:         business/python
URL:         https://raw.githubusercontent.com/ScoopInstaller/Main/master/scripts/python/install-pep-514.reg
First bytes: 57 69 6E 64 6F 77 73 20
Expected:    c2611c8010979a47cdeea1af17d5061d8df40c5356fd8c2f3ba9492d516a99d1
Actual:      27c2d7e74442f356d54d29000379afd55078fb4ede00455fc012f81745e4bcac

### PR DESCRIPTION
**Pull request by BotServiceAutomation**
- Closes #86

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### python

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Review by alexchapar — APPROVED** [review]

**Pull request merged**
